src/pages/home/components/HomeLandingImage.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useMediaQuery } from "react-responsive";
+import { Button } from "../../../shared/components/button/Button";
+import { LandingImage } from "shared/components/landing-image/LandingImage";
+import Gisau from "../../../assets/gisau-logo/gisau.png";
+
+export const HomeLandingImage = () => {
+  const isMobileView = useMediaQuery({
+    query: "(max-width: 639px)",
+  });
+
+  const handleClickButton = () => {
+    window.open("https://forms.gle/qujebG19m2VJzvBB6", "_blank", "noreferrer");
+  };
+
+  return (
+    <div>
+      <LandingImage>
+        <img
+          src={Gisau}
+          alt="Gisau Logo"
+          className="w-[30%] md:w-1/4 lg:w-1/5 m-auto"
+        />
+        <h1 className="text-white my-4 md:my-8 mx-2 font-proxima-nova text-center font-semibold text-xl sm:text-2xl md:text-4xl lg:text-[2.5rem] lg:leading-12">
```

Comment:
Can we make the `<h1>` generic in the `LandingImage` component? So you just need to pass the title's value from here.

Reply:
Some of the styling are different between both. I don't see any purpose of doing that for now. Maybe if its more common in the future, we can make it happen.